src/main/antlr4/imports/mysql_literal_tokens.g4

```diff
@@ -12,6 +12,7 @@ INT8: I N T '8';
 ACTION: A C T I O N;
 ADD: A D D;
 AFTER: A F T E R;
+ALGORITHM: A L G O R I T M;
```

Comment:
this file is generated by the script at src/main/antlr4/imports/generate_tokens.rb.  Can you add the tokens in there?  I'll update it to add a note so others don't fall into the same trap :)